services/coingecko.ts

```diff
@@ -56,13 +56,13 @@ const getCoinInfoByContract = async (search: string): Promise<TokenInfo> => {
   if (!publicRuntimeConfig.enableCoinGecko)
     return {prices: {}} as any; // eslint-disable-line
 
-  const storage = new WinStorage(`coingecko:${search.toLowerCase()}`);
+  const storage = new WinStorage(`coingecko:${search?.toLowerCase()}`);
```

Comment:
`search` is a mandatory argument, if it's null there's some bug on the thing calling this function. Either that, or the "if value" should be made _on the function that calls this one_.